internal/pb/export/export.proto

```diff
@@ -67,24 +76,44 @@ message TemporaryExposureKey {
   optional bytes key_data = 1;
 
   // Varying risks associated with exposure depending on type of verification
-  optional int32 transmission_risk_level = 2;
+  // Ignored by the v1.5 API when report_type is set.
+  optional int32 transmission_risk_level = 2 [deprecated = true];
 
   // The interval number since epoch for which a key starts
   optional int32 rolling_start_interval_number = 3;
 
   // Increments of 10 minutes describing how long a key is valid
-  optional int32 rolling_period = 4
-      [default = 144];  // defaults to 24 hours
+  optional int32 rolling_period = 4 [default = 144];  // defaults to 24 hours
+
+  // Data type representing why this key was published.
+  enum ReportType {
+    UNKNOWN = 0;  // Never returned by the API.
+    CONFIRMED_TEST = 1;
+    CONFIRMED_CLINICAL_DIAGNOSIS = 2;
+    SELF_REPORT = 3;
+    RECURSIVE = 4;
+    REVOKED = 5;  // Used to revoke a key, never returned by API.
+  }
+
+  // Type of diagnosis associated with a key.
+  optional ReportType report_type = 5;
+
+  // Number of days elapsed between symptom onset and the TEK being used.
+  // E.g. 2 means TEK is 2 days after onset of symptoms.
+  optional sint32 days_since_onset_of_symptoms = 6;
 }
 
 message TEKSignatureList {
   repeated TEKSignature signatures = 1;
 }
 
 message TEKSignature {
-  // Info about the signing key, version, algorithm, etc
+  // Info about the signing key, version, algorithm, etc. Only the
+  // verification_key_id, verification_key_version, and
+  // signature_algorithm fields within signature_info are read.
   optional SignatureInfo signature_info = 1;
-  // E.g., Batch 2 of 10
+  // E.g., Batch 2 of 10 - these fields are ignored on android in favor of the
```

Comment:
I'd revert these comments as well and leave our implementation some flexibility.

Reply:
done